packages/lesswrong/server/migrations/2019-05-14-fixCapitalizedSlugs.js

```diff
@@ -0,0 +1,30 @@
+import { Utils } from 'meteor/vulcan:lib';
+import Users from 'meteor/vulcan:users';
+import { registerMigration, migrateDocuments } from './migrationUtils';
+
+registerMigration({
+  name: "fixCapitalizedSlugs",
+  idempotent: true,
+  action: async () => await migrateDocuments({
+    description: `lowercase slugs`,
+    collection: Users,
+    unmigratedDocumentQuery: {
+      slug: /.*[A-Z].*/,
+    },
+    batchSize: 1000,
+    migrate: async (users) => {
+      let updates = _.map(users, user => ({
+        updateOne: {
+          filter: { _id: user._id },
+          update: {
+            $set: {
+              slug: Utils.slugify(user.slug)
```

Comment:
This potentially results in a user with a duplicate slug, which would be bad. There's an example in `vulcan-users/lib/modules/schema.js` of how to get a slug that's guaranteed to be unique (other than a race condition that won't come up in practice).

```suggestion
              slug: Utils.getUnusedSlugByCollectionName('Users', Utils.slugify(user.slug))
```